db/migrate/20160714133426_rename_arbitration_defaults_to_arbitration_profiles.rb

```diff
@@ -0,0 +1,15 @@
+class RenameArbitrationDefaultsToArbitrationProfiles < ActiveRecord::Migration[5.0]
+  def up
+    rename_table :arbitration_defaults, :arbitration_profiles
+    add_column :arbitration_profiles, :name, :string
+    add_column :arbitration_profiles, :description, :text
+    add_column :arbitration_profiles, :default_profile, :boolean, :default => false
```

Comment:
Right...we don't do SQL level default values.  Alternate is to use `default_value_for` in the active record model